### PR TITLE
Fix typos in `src/lib/OpenEXR/` 

### PR DESCRIPTION
**Pull request by luzpaz**
Fix typos in source comments and doxygen content. Found via `codespell -q 3 -S CHANGES.md,./ASWF/tsc-meetings -L dout,nnumber,offsetp`

### CODE REVIEW

File: src/lib/OpenEXR/ImfFastHuf.cpp [hunk 1]
```diff
@@ -507,7 +507,7 @@ FastHufDecoder::refill (
     //
     // We can have cases where the previous shift of bufferBack is << 64 -
     // this is an undefined operation but tends to create just zeroes.
-    // so if we won't have any bits left, zero out bufferBack insetad of computing the shift
+    // so if we won't have any bits left, zero out bufferBack insread of computing the shift
```
Comment (meshula) [inline src/lib/OpenEXR/ImfFastHuf.cpp]:
```suggestion
    // so if we won't have any bits left, zero out bufferBack instead of computing the shift
```

**Review by meshula — APPROVED** [review]
Thanks for the spell check!

**Comment by xlietz** [comment]
@luzpaz this commit needs to be properly signed off in order to merge. Directions on how to fix this are under "Some checks were not successful" - scroll down to the DCO line then click on the "Details" link to the right. Thanks!

**Comment by luzpaz** [comment]
Squashed and signed commit.

**Comment by cary-ilm** [comment]
Thanks, good catches. A little embarrassing, actually.

**Pull request merged**